samples/dashboards/html-component/nested-elements/demo.html

```diff
@@ -0,0 +1,2 @@
+<script src="https://code.highcharts.com/dashboards.js"></script>
```

Comment:
Please pull the latest changes from the `master` and update that path to:
 `<script src="https://code.highcharts.com/dashboards/dashboards.js"></script>`